src/client/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -79,6 +82,11 @@ private void ValidateScopeInput(ISet<string> scopesToValidate)
         public async Task<AuthenticationResult> RunAsync(CancellationToken cancellationToken = default)
         {
             Stopwatch sw = Stopwatch.StartNew();
+            activity = OpenTelemetryClient.AcquireTokenActivity.StartActivity("Token Acquisition", ActivityKind.Internal);
```

Comment:
TBD: We'd need to prefix activity/counter names and also satisfy schema requirements 
_Name must use alphanumeric characters ([a-zA-Z0-9]) and must start with a capital letter ([A-Z])._

Reply:
same for counters, e.g. acquire_token_success.

Reply:
Who would that be for? customers?
We would not want to send activities to the centralized telemetry. Only meters/histograms

Reply:
Yes, this will be for the customers. For example E&D if they want to capture this data from MSAL which is currently exposed using ITelemetryClient.